src/components/DevDummyLogin/index.js

```diff
@@ -58,13 +58,13 @@ const S = StyleSheet.create({
   },
   divider: {
     width: '100%',
-    backgroundColor: COLORS.blackgrey,
+    backgroundColor: COLORS.gray400,
     height: 2
   },
   passwordTextInput: {
     margin: 16,
     padding: 16,
-    backgroundColor: COLORS.lightgrey
+    backgroundColor: COLORS.gray100
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made to the `passwordTextInput` style are also acceptable if they meet the new design specifications. The padding and margin remain the same, but the background color has been changed from `COLORS.lightgrey` to `COLORS.gray100`. Again, make sure this color change doesn't negatively impact readability or accessibility.


<!-- This is an auto-generated comment by OSS CodeRabbit -->